drizzlepac/hlautils/sourcelist_generation.py

```diff
@@ -29,575 +22,74 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 
-def add_header_phot_tab(phot_table, drz_image, param_dict):
-    """
-    This task will make a header for a DAOPhot table based on information contained in the header of the 'drz_image'.
-
-    Tested.
+def create_dao_like_coordlists(fitsfile):
+    """Make daofind-like coordinate lists
 
     Parameters
     ----------
-    phot_table : string
-        DAOPhot sourcelist filename.
-
-    drz_image : string
-        filename of drizzled image whose header information will be used to populate the DAOPhot table header.
-
-    param_dict : dictionary
-        dictionary of drizzle, source finding and photometric parameters
+    fitsfile : string
+        Name of the drizzle-combined filter product to used to generate photometric sourcelists.
 
     Returns
     -------
-    nothing.
+    sources : astropy table
+        Table containing x, y coordinates of identified sources
     """
-    loaded_fits = fits.open(drz_image)
-
-    # -------------------------------
-    # FITS-compliant UTC time string
-    # -------------------------------
-    fits_string_date = datetime.datetime.utcnow().strftime("%A-%m-%dT%H:%M:%S")
-
-    # ---------------------------------
-    # Human-friendly local time string
-    # ---------------------------------
-    string_date = datetime.datetime.now().strftime("%A %B %d %H:%M:%S %Y")
-    fname = extract_name(drz_image)
+    hdulist = fits.open(fitsfile)
+    image = hdulist['SCI'].data
+    image -= np.nanmedian(image)
+    bkg_sigma = mad_std(image,ignore_nan = True)
+    daofind = DAOStarFinder(fwhm=3.5,threshold=2. * bkg_sigma)
```

Comment:
This should, even here, use a user-specified parameter value for the 'fwhm' parameter to allow the code to work accurately with data from different instruments.

Reply:
issue resolved in commit [648d4d1](https://github.com/spacetelescope/drizzlepac/pull/393/commits/648d4d1a1ef12c0f7200d13f526976f04996ff01)